### DISCUSSION TITLE
Which of the PrimeNG templates will not receive a v17 update?

### DISCUSSION THREAD

**Discussion by teokosmo** [primefaces/community#858] [PrimeNG Templates] [answered]
Hello,

Based on the answers in the following `discussion` urls I would like to know which of the PrimeNG templates, listed in https://primeng.org/templates, are considered legacy templates and thus will not receive a v17 update?

1. https://github.com/orgs/primefaces/discussions/771
2. https://github.com/orgs/primefaces/discussions/764

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

Templates below are the next-gen templates and will receive updates.

- Apollo
- Ultima
- Diamond
- Verona
- Avalon
- Freya
- Atlantis
- Sakai

Templates below are not next gen but rework is in progress and they also will receive an update

- California (updated to v17, will receive rework in the future)
- Roma (updated to v17, will receive rework in the future)
- Babylon (updated to v17, will receive rework in the future)
- Poseidon (update and structural rework is in progress, will receive a rework in the future)

For other templates, we don't have plans to update yet. But we're preparing a manual update guide for the users. They don't receive anything new but we'll provide a "how to" guide for them, guide will be available in upcoming weeks in primeng.org/guides

**Comment by teokosmo** [comment]
@cetincakiroglu 
Thanks for your detailed answer!
